logbook-netty/src/test/java/org/zalando/logbook/netty/IgnoringTest.java

```diff
@@ -0,0 +1,41 @@
+package org.zalando.logbook.netty;
+
+import io.netty.handler.codec.http.DefaultHttpContent;
+import io.netty.handler.codec.http.DefaultHttpRequest;
+import io.netty.handler.codec.http.HttpRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import static io.netty.buffer.Unpooled.wrappedBuffer;
+import static io.netty.handler.codec.http.HttpMethod.GET;
+import static io.netty.handler.codec.http.HttpVersion.HTTP_1_1;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+final class IgnoringTest {
+
+    final AtomicReference<State> state =
+            new AtomicReference<>(new Unbuffered());
+
+    final HttpRequest request = new DefaultHttpRequest(HTTP_1_1, GET, "/");
+
+    @Test
+    void shouldKeepBufferingWhenIgnoring() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/whiskeysierra/logbook/pullRequest?prid=4982439)